k8s/upgrade/src/bin/upgrade-job/helm/values.rs

```diff
@@ -361,6 +355,30 @@ where
             target_values.promtail_readiness_probe_http_get_path(),
             upgrade_values_file.path(),
         )?;
+
+        // This helm value key was changed:
+        // Ref: https://github.com/openebs/mayastor-extensions/pull/419
+        yq.set_literal_value(
+            YamlKey::try_from(".base.logging.silenceLevel")?,
+            source_values.deprecated_log_silence_level(),
+            upgrade_values_file.path(),
+        )?;
+        yq.delete_object(
+            YamlKey::try_from(".base.logSilenceLevel")?,
+            upgrade_values_file.path(),
+        )?;
+
+        // This is a fix for a typo in the .csi.node.pluginMounthPath key.
+        // It was fixed, and the key now is called .csi.node.pluginMountPath.
+        yq.set_literal_value(
+            YamlKey::try_from(".csi.node.pluginMountPath")?,
```

Comment:
@Abhinandan-Purkait -- one uses the value from the MounthPath in the MountPath key.

Reply:
Okay